torch/csrc/jit/ir/irparser.cpp

```diff
@@ -545,6 +600,18 @@ void IRParser::parse() {
     auto t = n->t_(attr::value, at::empty_strided(*sizes, *strides, options));
     (void)t;
   }
+
+  for (Node* n : deferred_empty_container_initializations_) {
```

Comment:
I don't think we're ever adding to this list

Reply:
whoops - thanks for catching that!